pom.xml

```diff
@@ -187,9 +187,9 @@ flexible messaging model and an intuitive client API.</description>
     <guava.version>31.0.1-jre</guava.version>
     <jcip.version>1.0</jcip.version>
     <prometheus-jmx.version>0.16.1</prometheus-jmx.version>
-    <confluent.version>7.0.1</confluent.version>
-    <kafka.confluent.schemaregistryclient.version>5.3.0</kafka.confluent.schemaregistryclient.version>
-    <kafka.confluent.avroserializer.version>5.3.0</kafka.confluent.avroserializer.version>
+    <confluent.version>6.2.8</confluent.version>
+    <kafka.confluent.schemaregistryclient.version>${confluent.version}</kafka.confluent.schemaregistryclient.version>
+    <kafka.confluent.avroserializer.version>${confluent.version}</kafka.confluent.avroserializer.version>
```

Comment:
Is it worth removing these variables entirely and replacing them with `confluent.version`?

Reply:
I guess so. I'll wait for options from other reviewer before making the change. @dlg99 What do you recommend?

Reply:
TBH I don't know why these are different.

Reply:
FYI the integ tests use `v6.0.1` - not sure if it makes sense to reconcile their version in this PR but we are not testing against 6.2.x atm: https://github.com/apache/pulsar/blob/4e29a1e9fc746d1b348dfe25ca047f0effd93588/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/AvroKafkaSourceTest.java#L483

Reply:
Thanks for the review feedback @dlg99 and @aymkhalil . PTAL